packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -1893,6 +1894,24 @@ export type Profile = {
     displayName?: string;
 };
 
+// @beta
+export interface PromptStrings {
```

Comment:
My assumption here is that this prompt could be used for many different purposes and not just spotlight in the future?
However, we have other kinds of prompts already not following this pattern - captions language, you're muted notification.
(Open question) Maybe we shouldn't nest spolight inside prompt and just have `spotlightPrompt`?

Reply:
Yes, I was trying to leave the Prompt component reusable for the future even though we have some prompts that have their own components. I'll leave it as is since it is an internal component and maybe another prompt might be needed in the future.
Nesting one level into spotlightPrompt works for me! Thanks.